git-refresher/moving-between-branches.qmd

```diff
@@ -1,18 +1,61 @@
---- 
+---
 title: "Moving between branches"
 ---
 
-...
+This section will guide you through switching branches in your repo.
+
+Switching branches is useful for when we might need to pause our work on one branch and continue working on an issue in another branch. Before you switch branches it is important to commit and push the changes in the branch you're currently in. It's also important to pull changes after you switch to your new branch.
 
 ::: panel-tabset
 ## Git Bash
 
+First you need to update your local repo with the changes made to the remote one. This ensures that you have a full list of remote branches and the latest changes in the code to avoid conflicts. Enter the command below:
+
+`git pull`
+
+In Git Bash, you can switch branches using either of the following options:
+
+`git checkout`
+
+OR
+
+`git switch`
+
 ## GitHub Desktop
 
 ## VS Code
 
 ## RStudio
 
+-   Go to the right hand side of RStudio and click on the Git tab.
+
+-   Click on the downwards arrow to "pull" the latest changes from the remote repo. This ensures that:
+
+    -   you have a full list of remote branches.
+    -   you have the latest changes in the code to avoid conflicts.
+
+-   Click on the drop down menu of branches. Usually "main" is the default branch shown before switching.
+
+-   Click on `<github_id>_main`
```

Comment:
At this point in the work through, they should really be on `<github_id>/main` already, so I think the steps here would again be: switch back to main, see your changes have been undone, then switch back to your own main branch.